### DISCUSSION TITLE
Task Feedback

### DISCUSSION THREAD

**Discussion by LesterCerioli** [LesterCerioli/Express-Store#42] [General]
Good Morning!!

    I'm almost finishing the development of the Core context Infrastructure project. Then I intend to start the development of the microservice itself. The idea is to migrate legacy SimplCommerceCore to ExpressStoreCore. Another note is that the API is being built in a NET Core API template and thus it will be published to AWS Lambda.

    I also wrote the API gateway framework and it was published on dev-p1

**Comment by MadalenaDiasSec** [comment]
Good Morning @LesterCerioli 
I keep creating the basic structures of the microservices and the Infrastructure structure, I think it will make it much easier.

**Comment by MadalenaDiasSec** [comment]
Thank you very much and I wish the same and that next year we will be very successful and that this project will grow more and more

**Comment by MadalenaDiasSec** [comment]
Dear I'm evaluating the table mapping study because we decided to restructure using hexagonal architecture. In principle we will use a Sql database but we will continue with isolated microservices fronts so that in the future we can decide for scalability.

We believe that at this moment it is necessary to raise the platform so that it is feasible for users to use it.
​
    At the same time, we will be working on scalability with isolated bases and services

**Comment by MadalenaDiasSec** [comment]
@LesterCerioli Is it possible you require pull-request for dev-p1?
I need  use new architecture implemented by you needing to use your implementation as a base.

**Comment by LesterCerioli** [comment]
okay @MadalenaDiasSec  I'll merge to dev-p1.
